manifests/s/Samsung/TouchLogon/2.0.0/Samsung.TouchLogon.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.TouchLogon
+PackageVersion: 2.0.0
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=4590954&CDCttType=SW&ModelType=C&ModelName=XE700T1A-A02US&VPath=SW/201201/20120105111636324/Touch_Logon_64bit_2.0.0.8.ZIP
+  InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: msi
+  NestedInstallerFiles:
+  - RelativeFilePath: Setup_x64-R.msi
+  InstallerSha256: 65513A4FBB2C84DF813A5DAAE3716F4BA18F56F5AFD8982EAA833BFFC394173D
+  ProductCode: '{AFF0220E-1F50-4B46-91BF-7812F71275F0}'

```

```suggestion
  ProductCode: '{AFF0220E-1F50-4B46-91BF-7812F71275F0}'
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/201201/20120105111636324/Touch_Logon_64bit_2.0.0.8.ZIP
  InstallerLocale: en-US
  Architecture: x86
  InstallerType: zip
  NestedInstallerType: msi
  NestedInstallerFiles:
  - RelativeFilePath: Setup_x86-R.msi
  InstallerSha256: 65513A4FBB2C84DF813A5DAAE3716F4BA18F56F5AFD8982EAA833BFFC394173D
  ProductCode: '{D1190F96-AF15-47B1-BCFD-F6EAE2D91EDE}'
```
Please also add the x86 installer